### PR TITLE
[QWIN][PROFILE-14] Add connections drawer for the Welcome Back widget

### PR DESCRIPTION
**Pull request by mdiha**
I added connections drawer for the Welcome Back widget.

### CODE REVIEW

File: portlets/src/main/frontend/src/apps/profileStats/components/ConnectionsDrawer.vue [hunk 1]
```diff
@@ -0,0 +1,157 @@
+<template>
+  <v-navigation-drawer
+    v-model="connectionsDrawer"
+    absolute
+    right
+    stateless
+    temporary
+    width="420"
+    max-width="100vw"
+    max-height="100vh"
```
Comment (ahamdi) [inline portlets/src/main/frontend/src/apps/profileStats/components/ConnectionsDrawer.vue]: Could you avoid putting styles inside the DOM and use CSS classes.

File: portlets/src/main/frontend/src/apps/profileStats/components/ConnectionsDrawer.vue [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+      this.initConnections();
+    },
+      methods: {
+      closeDrawer() {
+        this.$emit('closeDrawer');  
+      },
+      initConnections() {
+        getUserConnections().then(data => {
+          this.connections = data.users;
+          console.log('### connections: ', this.connections);
```
Comment (ahamdi) [inline portlets/src/main/frontend/src/apps/profileStats/components/ConnectionsDrawer.vue]: Remove this line pelase

File: portlets/src/main/frontend/src/apps/profileStats/components/ConnectionsDrawer.vue [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+          return this.connections.filter(item => item.fullname.toLowerCase().match(this.search.toLowerCase())); 
+        } else {
+          return this.connections;
+        }
+      },
+      showConnectionRequests() {
+        return this.connectionRequests > 0;
+      },
+      showConnection() {
+        console.log('changed !');
```
Comment (ahamdi) [inline portlets/src/main/frontend/src/apps/profileStats/components/ConnectionsDrawer.vue]: Remove console logs

**Review by ahamdi — CHANGES_REQUESTED** [review]

File: portlets/src/main/frontend/src/apps/profileStats/components/ConnectionsDrawer.vue (hunk 1 above)
Comment (mdiha) [reply portlets/src/main/frontend/src/apps/profileStats/components/ConnectionsDrawer.vue]: Okay

File: portlets/src/main/frontend/src/apps/profileStats/components/ConnectionsDrawer.vue (hunk 2 above)
Comment (mdiha) [reply portlets/src/main/frontend/src/apps/profileStats/components/ConnectionsDrawer.vue]: Okay

File: portlets/src/main/frontend/src/apps/profileStats/components/ConnectionsDrawer.vue (hunk 3 above)
Comment (mdiha) [reply portlets/src/main/frontend/src/apps/profileStats/components/ConnectionsDrawer.vue]: Okay

**Review by ahamdi — APPROVED** [review]
LGTM

**Pull request merged**